app/controllers/unattended_controller.rb

```diff
@@ -67,6 +67,12 @@ def pxe_config
   # Using alias_method causes test failures as iPXE method is unknown in an empty DB
   def gPXE; iPXE; end
 
+  protected
+
+  def require_ssl?
```

Comment:
This also isn't equivalent, the previous code only skipped the SSL filter on certain actions and when there's no spoof or hostname parameter.